x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/DefaultEndPointsIT.java

```diff
@@ -110,4 +116,39 @@ private static void assertDefaultE5Config(Map<String, Object> modelConfig) {
             Matchers.is(Map.of("enabled", true, "min_number_of_allocations", 0, "max_number_of_allocations", 32))
         );
     }
+
+    public void testMultipleInferencesTiggeringDownloadAndDeploy() throws InterruptedException {
+        assumeTrue("Default config requires a feature flag", DefaultElserFeatureFlag.isEnabled());
```

Comment:
This isn't testing that there's only one single download task, is it? Would it be hard to add such check?